pkg/updater/updater.go

```diff
@@ -81,6 +82,18 @@ func Bootstrap(ctx context.Context, pkg string) error {
 	return u.Bootstrap(ctx, pkg)
 }
 
+// Purge removes files installed by the updater
+func Purge() {
+	// golanci on windows somehow expects a return value
+	service.RemoveAgentUnits() // nolint:errcheck
```

Comment:
On windows [RemoveAgentUnits](https://github.com/DataDog/datadog-agent/blob/5a2a21c211c1b735fab7c3294f1247eeb9169368/pkg/updater/service/datadog_agent_windows.go#L27) has an error as a return value, which is why the CI triggered. Let's remove it and have the same signature

Reply:
Linux has an error as the return value as well. Let's just log the error, we want to be able to debug this in e2e and with customers

Reply:
there's no error on linux, thanks for catching the windows
On purpose I'm passing through each command to cleanup everything that is possible, and log warns for each error seen
https://github.com/DataDog/datadog-agent/pull/23597/files#diff-105824131aeb51aea4c79cc098a114e7346fbbaeb2d709e24ac878eedc596e4eR80-R112